CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/editors/DateTimeEditorFactory.java

```diff
@@ -73,6 +73,8 @@ public String getAttributeType() {
         private static final int MILLIS_SPINNER_WIDTH = 60;
         private static final int NANOSECONDS_IN_MILLISECOND = 1000000;
         private static final int MILLISECONDS_IN_SECOND = 1000;
+        
+        private static final String LABEL = "label";
```

Comment:
This seems like a poor id value.

Reply:
Don't think you're wrong. Doesn't seem to make sense with respect to its context. Have renamed to LABEL_ID and the value can probably be addressed another time